src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -1268,6 +1283,24 @@ internal string EscapePath(string path, bool ignoreSpaces = false)
             return path;
         }
 
+        internal bool UseUnixPathSeparators
+        {
+            get
+            {
+                if (PlatformUtilities.IsWindows())
```

Comment:
I combined them for Andrew. I didn't care either way and as long as the Windows check is first, then I think it will be clearer?